plop_templates/test_template.txt

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
-import sinon from 'sinon';
+//import sinon from 'sinon';
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
 
 import {{properCase name}} from '../index';
 
 describe('{{properCase name}}', () => {
-  let defaultProps;
+  // let defaultProps;
```

Comment:
I agree, which is why I commented them out. I want the first test template to not immediately fail linting.